toggle/pkg_resources

```diff
@@ -0,0 +1,610 @@
+# This file helps to compute a version number in source trees obtained from
```

Comment:
Does this need to be checked in or does Versioneer generate this on demand?

If this does need to be checked it, can it be checked in separately from all the 2to3 changes?

Reply:
It looks like you added and then removed Versioneer and this was left over?